src/cheri_insts.sail

```diff
@@ -1305,13 +1305,40 @@ function handle_load_data_via_cap(rd, auth_idx, auth_val, vaddrBits, is_unsigned
 }
 
 union clause ast = LoadDataDDC : (regidx, regidx, bool, word_width)
+/*!
+ * Integer register *rd* is replaced with the signed or unsigned byte,
```

Comment:
It seems redundant to list all this in detail here when it's just a call through to LoadDataCap, which already has the description. By no means the end of the world though.

Reply:
It calls the helper, not LoadDataCap. Those helpers don't have any saildoc, nor is any additional documentation given to them in cheri-architecture. Also they appear in an earlier section, whereas this description goes on the actual instruction specification page(s), so I think there is value in detailing everything there even if the handle functions were also documented.

Reply:
Oh yes, sorry, misread. It still seems like ideally these would be refactored to avoid the redundancy, but I can't see directly an elegant way to achieve that.

Reply:
Yeah I think that's too much magic to get it to work with cheri-architecture's layout. Especially given we also have the mode-dependent instructions.